src/main/java/core/basesyntax/file/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file at " + filePath,e);
```

Comment:
Throwing a `RuntimeException` for an `IOException` might not be the best approach as it can terminate the application abruptly. Consider using a custom exception or handling the error more gracefully to provide better feedback to the user.